src/frontend/src/lib/components/convert/ConvertFee.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import { BigNumber } from '@ethersproject/bignumber';
+	import ConvertAmountDisplay from '$lib/components/convert/ConvertAmountDisplay.svelte';
+	import { formatToken } from '$lib/utils/format.utils';
+
+	export let feeAmount: bigint | undefined = undefined;
+	export let symbol: string;
+	export let decimals: number;
+	export let exchangeRate: number | undefined = undefined;
+	export let zeroAmountLabel: string | undefined = undefined;
+
+	let formattedFeeAmount: number | undefined;
+	$: formattedFeeAmount = nonNullish(feeAmount)
+		? Number(
+				formatToken({
```

Comment:
can formatToken be used inside Number? I really don't know, I imagine you tested and it went all good?

(maybe a short unittest for this component, just for this part of logic?)

Reply:
If you are sure that i works (tested manually) and are tight with time, maybe add a TODO?

Or even better, since it seems you are using `Number(formatToken(...))` more times, maybe a dedicated function with its own test (in another future PR)?

Reply:
Yep, all good, it can be used like this. This pattern is already in the codebase, e.g. `exhange.utils -> usdValue`. Regarding a util - yes, a good idea, I am indeed using it in a plenty of files across the conversion flow. Add todo comment for now.

Reply:
Tks for checking!